components/dashboard/src/projects/NewProject.tsx

```diff
@@ -411,7 +411,7 @@ function NewTeam(props: {
 
 async function openReconfigureWindow(params: { account?: string, onSuccess: (p: any) => void }) {
     const { account, onSuccess } = params;
-    const state = btoa(JSON.stringify({ from: "/reconfigure", next: "/new" }));
+    const state = btoa(JSON.stringify({ from: "/reconfigure", next: "/new" })); // TODO: `/new?team={team?.slug}` ?
```

Comment:
@AlexTugarev Please double-check my suspicion here:
- When creating a new project under a team, we already have a `team` (see `props.onSuccess(team)`)
- Here, we say that the "next" URL after `/reconfigure` should be `/new`

Should it be `/new?team={team.slug}` if there is a `team`?

(I'm trying to fix a weird UX problem where I did "open team page" > "add project" and then got asked "which team?" again, but I don't know what this "next" URL does yet)

Reply:
@jankeromnes, please ignore `state` param for the time being. It's presence is important for to support the old way marketplace installation _and_ the new project driven installations with the same app. 

> problem where I did "open team page" > "add project" and then got asked "which team?" again

how to reproduce this?

Reply:
> please ignore `state` param for the time being.

Sure, thanks! Will ignore for now.

> how to reproduce this?

It only happened to me once, but then I couldn't reproduce, so I tried to find the bug by looking at the code.

However, since it's hard to reproduce, I think it's fine to ignore it for now. Will remove it from this PR.